connaisseur/util.py

```diff
@@ -71,7 +71,7 @@ def get_admission_review(
         }
     """
     review = {
-        "apiVersion": "admission.k8s.io/v1beta1",
+        "apiVersion": "admission.k8s.io/v1",
```

Comment:
I think it is possible to specify that either `v1` or `v1beta1` is fine. @phbelitz what do you think?

Reply:
welp, depends on how many versions we want to support. if we say `latest - 5`, then we can savely get rid of all `v1beta`s, since it's deprecated since v1.16